src/lib.rs

```diff
@@ -420,3 +416,9 @@ static PRIV_VXP_METHODS: vmod_priv_methods = vmod_priv_methods {
     type_: c"VXP type".as_ptr(),
     fini: Some(vmod_priv::on_fini::<Vxp>),
 };
+
+/// Convert an i64 to a `usize`, clamping it between zero to the maximum value of usize
+pub(crate) fn clap_i64_to_usize(value: i64) -> usize {
```

Comment:
This should be "clap", not "clamp", methinks

Reply:
I managed to write the exact opposite of what I meant...

So, yes, this should be claMp, not clap

Reply:
ah, thx, fixed